source/renderer/app/components/sidebar/wallets/WalletSearch.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import React from 'react';
+import { intlShape, injectIntl, defineMessages } from 'react-intl';
+import { noop } from 'lodash/fp';
+import classNames from 'classnames';
+import { Input } from 'react-polymorph/lib/components/Input';
+import SVGInline from 'react-svg-inline';
```

Comment:
This package is deprecated and they suggest using [SVGR](https://github.com/gregberge/svgr).

In my opinion we could even create a simple wrapper around svg by hand and it works without any external dependencies.

I don't want to bring any confusion with this suggestion but let's discuss what is worth.
I see that `react-svg-inline` is not heavily used in our codebase so we might get rid of it easily (as it's deprecated).

Reply:
Good catch! I think this would fit better in a different PR as the current code base heavily relies on `react-svg-inline`. Would you like to create a tech debt ticket for this one?

Reply:
Sure :)

https://input-output.atlassian.net/browse/DDW-919